perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/MembersManagerBlImpl.java

```diff
@@ -1510,6 +1513,82 @@ public Member setStatus(PerunSession sess, Member member, Status status) throws
 		}
 	}
 
+	private void removeMemberFromParentVos(PerunSession sess, Member member ) {
+		User user = getPerunBl().getUsersManagerBl().getUserByMember(sess, member);
+		try {
+			Vo memberVo = getPerunBl().getVosManager().getVoById(sess, member.getVoId());
+			List<Vo> parentVos = getPerunBl().getVosManager().getParentVos(sess, member.getVoId());
```

Comment:
Here in the BL layer, we should usually call BL methods instead of entry layer methods because it is unnecessary to check privileges and other things the entry layer checks. Then, you can remove PrivilegeException from imports.